net/netns/netns.go

```diff
@@ -66,3 +68,20 @@ type Dialer interface {
 	Dial(network, address string) (net.Conn, error)
 	DialContext(ctx context.Context, network, address string) (net.Conn, error)
 }
+
+func IsLocalhost(host string) bool {
```

Comment:
does this need to be exported? looks like not?

If it does, though, then the bar for docs is raised and this needs proper docs.

Reply:
Unexported it.